packages/component-library/stories/Button.story.js

```diff
@@ -3,34 +3,103 @@ import React from "react";
 import { storiesOf } from "@storybook/react";
 import { action } from "@storybook/addon-actions";
 import { checkA11y } from "@storybook/addon-a11y";
-import { withKnobs, text } from "@storybook/addon-knobs";
+import {
+  withKnobs,
+  text,
+  color,
+  optionsKnob as options,
+  select
+} from "@storybook/addon-knobs";
+import notes from "./button.markdown.md";
 import { Button } from "../src";
 import { storybookStyles } from "./storyStyles.js";
 
-const displayName = Button.displayName || "Button";
-
-const description = `
-  This is some basic usage with the button with providing a label to show the text.
-  Clicking should trigger an action.`;
-
-const simpleButtonLabel = "Hello button";
-
 export default () =>
-  storiesOf("Component Lib|Basic UI Components/Button", module)
+  storiesOf("Component Lib|Basic Inputs/Button", module)
     .addDecorator(withKnobs)
     .addDecorator(checkA11y)
     .addDecorator(story => (
       <div style={storybookStyles.storyGrid}>
         <div style={storybookStyles.storyGridItem}>{story()}</div>
       </div>
     ))
-    .add("Simple usage", () => (
-      <Button onClick={action("clicked")}>{simpleButtonLabel}</Button>
-    ))
-    .add("With some emoji", () => (
-      <Button onClick={action("clicked")}>
-        <span role="img" aria-label="So cool">
-          😀 😎 👍 💯
-        </span>
-      </Button>
-    ));
+    .add(
+      "Standard",
+      () => {
+        const label = text("Label", "Hello CIVIC");
```

Comment:
Recommend pulling the knobs outside of the function block body so they can be reused across stories.

```
import { storybookStyles } from "./storyStyles.js";

// define knobs here so they can be reused across stories
const label = text("Label", "Hello CIVIC");
…

export default () =>
…

    .add(
      "Standard",
// can use concise syntax here
      () => (<Button onClick={action("clicked")}>{label}</Button>),
      { notes }
    )
```

Reply:
Note, in case anyone else is following along -- what I suggested doesn't work. Knobs need to be defined withing the function block as part of .add